lib-release.js

```diff
@@ -277,6 +279,7 @@ const confirmedRelease = (type, version) => {
     () => execaSeries([
       hasTestScript && 'npm run test',
       'npm run build',
+      processArgs.length ? `git add ./${processArgs.join(' ./')}`: '',
```

Comment:
I'm still concerned if we should commit build artefacts or not 🤔 
Though to be discussed somewhere else...

Reply:
totally agree. I will open a issue